ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-release-4.12.yaml

```diff
@@ -99,6 +105,53 @@ tests:
   steps:
     cluster_profile: vsphere
     workflow: openshift-e2e-vsphere-etcd-scaling
+- as: e2e-aws-periodic-pre
+  optional: true
+  run_if_changed: .*periodic.*
+  steps:
+    cluster_profile: aws
+    test:
+    - as: test
+      commands: |
+        make e2e-periodic
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+      timeout: 3h0m0s
+    workflow: ipi-aws
```

Comment:
This is a presubmit so not sure we need this one backported? We should only be testing/making changes to the E2E on the main branch right?

Reply:
We had a chat and decided to keep this to signal to inform us whether periodics can be successful at presubmit stage.